### PR TITLE
docs(style): fix non-standard spaces between letters

### PR DESCRIPTION
**Pull request by Xunzhuo**
### What this PR does / why we need it:

I spent hours of mine fixing all the non-standard spaces with wrong styles in zh docs, covering all the zh docs.

Add or remove spaces between letters in these fields:
+ spaces between zh and en letters
+ spaces between one line code block and words
+ spaces between numbers and words

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Could you add a linter to check this?

**Comment by Xunzhuo** [comment]
@spacewander Yes, I can try to start the other PR to do this later.

File: docs/zh/latest/FAQ.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ title: 常见问题
 
 ## APISIX 和其他的 API 网关有什么不同之处？
 
-APISIX 基于 etcd 来完成配置的保存和同步，而不是 postgres 或者 MySQL 这类关系型数据库。
+APISIX 基于 etcd 来完成配置的保存和同步，而不是 Postgre 或者 MySQL 这类关系型数据库。
```
Comment (moonming) [inline docs/zh/latest/FAQ.md:30]: typo: PostgreSQL

File: docs/zh/latest/plugins/consumer-restriction.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ HTTP/1.1 403 Forbidden
 
 ### 如何限制 `service_id`
 
-`service_id`方式需要与授权插件一起配合使用，这里以key-auth授权插件为例。
+`service_id` 方式需要与授权插件一起配合使用，这里以key-auth授权插件为例。
```
Comment (spacewander) [inline docs/zh/latest/plugins/consumer-restriction.md:121]:
```suggestion
`service_id` 方式需要与授权插件一起配合使用，这里以 key-auth 授权插件为例。
```

Comment (Xunzhuo) [reply docs/zh/latest/plugins/consumer-restriction.md]: Done.

File: docs/zh/latest/FAQ.md (hunk 1 above)
Comment (Xunzhuo) [reply docs/zh/latest/FAQ.md]: Done.

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**